.gitignore

```diff
@@ -21,7 +21,6 @@
 !.travis*
 !.tx
 !.husky
-*.css
```

Comment:
Let's revert changes to this file, remove `globals.css` and we're all set

Reply:
Done